simulation/do_nothing_plugin/include/do_nothing_plugin/plugin.hpp

```diff
@@ -74,13 +71,19 @@ public:                                                                \
 private:                                                               \
   TYPE FIELD_NAME;
   // clang-format off
-  DEFINE_GETTER_SETTER(CurrentTime,       double,                                                        current_time_)
-  DEFINE_GETTER_SETTER(StepTime,          double,                                                        step_time_)
-  DEFINE_GETTER_SETTER(EntityStatus,      std::shared_ptr<traffic_simulator::CanonicalizedEntityStatus>, entity_status_)
-  DEFINE_GETTER_SETTER(BehaviorParameter, traffic_simulator_msgs::msg::BehaviorParameter,                behavior_parameter_)
-  DEFINE_GETTER_SETTER(UpdatedStatus,     std::shared_ptr<traffic_simulator::CanonicalizedEntityStatus>, updated_status_)
+  DEFINE_GETTER_SETTER(BehaviorParameter,  traffic_simulator_msgs::msg::BehaviorParameter,                   behavior_parameter_)
+  DEFINE_GETTER_SETTER(CurrentTime,        double,                                                           current_time_)
+  DEFINE_GETTER_SETTER(EntityStatus,       std::shared_ptr<traffic_simulator::CanonicalizedEntityStatus>,    entity_status_)
+  DEFINE_GETTER_SETTER(HdMapUtils,         std::shared_ptr<hdmap_utils::HdMapUtils>,                         hdmap_utils_)
+  DEFINE_GETTER_SETTER(PolylineTrajectory, std::shared_ptr<traffic_simulator_msgs::msg::PolylineTrajectory>, polyline_trajectory)
+  DEFINE_GETTER_SETTER(Request,            traffic_simulator::behavior::Request,                             request)
+  DEFINE_GETTER_SETTER(StepTime,           double,                                                           step_time_)
+  DEFINE_GETTER_SETTER(UpdatedStatus,      std::shared_ptr<traffic_simulator::CanonicalizedEntityStatus>,    updated_status_)
   // clang-format on
 #undef DEFINE_GETTER_SETTER
+
+  void checkPolylineTrajectory();
+  void followPolylineTrajectory();
```

Comment:
These functions are for implementing DoNothingBehavior::update, so they do not need to be member functions. It is sufficient that they are local functions in the .cpp file.